source/guides/administrator.rst

```diff
@@ -61,6 +61,7 @@ Deployment
    /deployment/sso-gitlab*
    /deployment/sso-google*
    /deployment/sso-office*
+   /deployment/saml-faq*
```

Comment:
Propose we don't add this separately to the sidebar, given we don't do that for other FAQs. Including it in the individual SAML pages is sufficient.

Reply:
Removed.